internal/service/rds/cluster_test.go

```diff
@@ -2193,6 +2202,40 @@ func TestAccRDSCluster_enableHTTPEndpoint(t *testing.T) {
 	})
 }
 
+func TestAccRDSCluster_MigrateFromPluginSDK(t *testing.T) {
+	ctx := acctest.Context(t)
+	resourceName := "aws_rds_cluster.test"
+	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
+
+	var dbCluster rds.DBCluster
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acctest.PreCheck(t) },
+		ErrorCheck:   acctest.ErrorCheck(t, rds.EndpointsID),
+		CheckDestroy: testAccCheckClusterDestroy(ctx),
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"aws": {
+						Source:            "hashicorp/aws",
+						VersionConstraint: "4.51.0",
```

Comment:
Would it make sense to update this to the last version where we had an SDK-based resource? Probably `4.54.0`, unless this goes in today